roles/download/tasks/kubeadm_images.yml

```diff
@@ -0,0 +1,35 @@
+- name: gets the kubeadm version
+  command: "{{ local_release_dir }}/kubeadm version -o short"
+  register: kubeadm_output
+
+- name: sets kubeadm api version to v1alpha1
```

Comment:
I had this thought about the kubeadm role already, but maybe a single task that sets this fact with multiple jinja conditions could be better ?

Reply:
thanks,  i wiil do that.

Reply:
Actually I thought about setting the fact and use it the same way in templates but that's fine